jenkins-scripts/dsl/sdformat.dsl

```diff
@@ -4,7 +4,7 @@ import javaposse.jobdsl.dsl.Job
 def sdformat_supported_branches = [ 'sdformat4', 'sdformat6', 'sdformat8' , 'sdformat9', 'sdformat10' ]
 def sdformat_gz11_branches = [ 'sdformat8', 'sdformat9', 'sdformat10', 'master' ]
 // nightly and prereleases
-def extra_sdformat_debbuilder = [  ]
+def extra_sdformat_debbuilder = [ 'main' ]
```

Comment:
uh, this is going to create `main-debbuilder` job which is something we probably don't want. Which was your intention here?

Reply:
Oh! I was trying to create an `sdformat11-debbuilder`, from the `main` branch :upside_down_face:

Reply:
Reverted in f720f98